images/microOS/config.xml

```diff
@@ -580,6 +580,7 @@
         <package name="python3-aiofiles"/>
         <package name="python3-requests"/>
         <package name="python3-rados"/>
+        <package name="python3-websockets"/>
```

Comment:
Why is this required? Aren't we installing the dependencies from the requirements.txt? (I'm making this assumption based on things like fastapi missing).

Reply:
@jhesketh that's question is not to me, and since it is not related to this particular patch in order to not block this PR I suggest you discuss this with @jecluis, @tserong, and the rest of the team on our slack channel.

Reply:
@kshtsk why is it in this pull request though?

Reply:
yeah, lets drop it from this PR and have it merged with #269

Reply:
> Why is this required? Aren't we installing the dependencies from the requirements.txt? (I'm making this assumption based on things like fastapi missing).

Actually some of this stuff is a little bit messy ATM -- images/microOS/config.sh is doing `pip install fastapi uvicorn` rather than using requirements.txt. I'm trying to clean this up as part of #91

Reply:
I don't understand why do we continue discussing it in this PR where the change already dropped.

Reply:
Josh asked a question and I thought it might be helpful to answer it.